cluster.yml

```diff
@@ -12,6 +12,8 @@
         - check
   vars:
     ansible_connection: local
+    # TODO aanno: where is the right place for this?
+    kubelet_cgroup_driver: systemd
```

Comment:
This is probably the wrong place to set the variable. But what would be the right place?

Reply:
You offer very bad things.

Assigning variables to a playbook is bad practice
Changing the script so that it works only on one os - even worse

I think you should better study how the ensemble works and works.

Reply:
This is _not_ in the change set any more (use 'Files changed' tab). 

BTW, I _asked_ were the right place to put that would be...